organization/views.py

```diff
@@ -184,6 +184,25 @@ def get_success_url(self):
         uuid = self.kwargs[self.slug_field_kwarg]
         return reverse('view-organization', kwargs={'slug': uuid})
 
+    def get_cancel_url(self):
+        return reverse('view-organization', kwargs={'slug': self.kwargs['organization_id']})
+
+
+class OrganizationCreateView(BaseCreateView):
+
+    def get_cancel_url(self):
+        return reverse('view-organization', kwargs={'slug': self.kwargs['organization_id']})
+
+
+class OrganizationDeleteRelationshipView(BaseDeleteRelationshipView):
+
+    def get_cancel_url(self):
+        organization_id = self.kwargs['organization_id']
+        pk = self.kwargs['pk']
+        return reverse('edit-organization-composition',
```

Comment:
Looks like this will take the user to the `EditComposition` view no matter what entity type they were trying to delete. I think we could generalize it if we made use of the `model` attribute on the class to format the URL:

```suggestion
        return reverse('edit-organization-{}'.format(self.model.__name__.lower(),
```

Reply:
Good catch! This is clever, however the `MembershipPerson` model doesn't follow the same template naming convention. It expects `edit-organization-membership`. Do you think we should override the method for `OrganizationDeleteMembershipView`?

Reply:
Ugh, true! Yeah, I think in that case overriding it is fine.